dotnet/src/Planners/Planners.Handlebars/Handlebars/HandlebarsPlanner.cs

```diff
@@ -25,7 +25,7 @@ public sealed class HandlebarsPlanner
     public static readonly HandlebarsPromptTemplateOptions PromptTemplateOptions = new()
     {
         // Options for built-in Handlebars helpers
-        Categories = new Category[] { Category.Math },
+        Categories = new Category[] { Category.DateTime },
```

Comment:
Is this category for the default helpers included with the 'Handlebars.Net' NuGet package, or is it for the SK default helpers?